recipes/r-ngsplot/build.sh

```diff
@@ -4,8 +4,7 @@ mkdir -p $outdir
 mkdir -p $PREFIX/bin
 cp -R * $outdir 
```

Comment:
I'm not sure if it's still the case, but I know `conda-build` sometimes wrote files to the cwd during building (I think `conda-build.sh` or `run-test.sh` or something). Those files will/might get copied too if you `cp -R *` here.